### PR TITLE
Added documentation on setting the broker.rack property

### PR DESCRIPTION
**Pull request by prestonmcgowan**
# Description

Provide documentation on how the broker.rack property can be included in a broker deployment.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

I have deployed out to an AWS cluster.

**Test Configuration**:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: hosts_example.yml [hunk 1]
```diff
@@ -165,13 +165,25 @@ kafka_broker:
       ## By default the first host will get broker id=1, second gets id=2. Set broker_id to customize
       # broker_id: 3
 
+      ## (optionally) Rack aware replication assignment for fault tolerance. Examples: `RACK1`, `us-east-1d`
```
Comment (domenicbove) [inline hosts_example.yml]: We should introduce this a little bit. "Properties can also be applied on a host by host basis. In the below example we configure a Multi Region Cluster by setting the broker.rack property on each host"

Sorry I don't mean to hijack this. I think the feature is called "multi region cluster"

Comment (domenicbove) [reply hosts_example.yml]: Thinking about it more dont we also need this one set for MRC?
replica.selector.class=org.apache.kafka.common.replica.RackAwareReplicaSelector

Might as well complete the loop and get that in somehow

Comment (prestonmcgowan) [reply hosts_example.yml]: I'm good with updating the `replica.selector.class` as well.
I will also update the language a bit to be more generic as you stated.

I will then test it out on my AWS cluster to ensure they are set accordingly.

Comment (prestonmcgowan) [reply hosts_example.yml]: NOTE: https://github.com/confluentinc/examples/blob/master/multiregion/README.md

**Comment by prestonmcgowan** [comment]
I updated the documentation as suggested.
I tested against my AWS cluster and witnessed the `replica.selector.class` setting being set.

File: hosts_example.yml [hunk 2]
```diff
@@ -165,13 +165,31 @@ kafka_broker:
       ## By default the first host will get broker id=1, second gets id=2. Set broker_id to customize
       # broker_id: 3
 
+      ## Properties can also be applied on a host by host basis.
+      # In the below example we configure a Multi-Region Replication by setting the following properties on each host:
+      # * broker.rack
```
Comment (domenicbove) [inline hosts_example.yml]: I'd prefer this to take less lines. Also the NOTE is unnecessary, ppl can (and should) read the MRC docs themselves

**Pull request merged**